src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -314,11 +314,13 @@ class WorkflowManagerActor(backend: Backend) extends LoggingFSM[WorkflowManagerS
       BackendCallKey(descriptor.namespace.workflow.findCallByName(callName).get, key.index, key.attempt)
     def backendCallFromKey(descriptor: WorkflowDescriptor, callName: String, key: ExecutionDatabaseKey) =
       backend.bindCall(descriptor, callKey(descriptor, callName, key))
+
+    import cromwell.engine.finalcall.FinalCall._
```

Comment:
I say this admittedly as someone who is biased against imports in the middle of code but this is now being imported in a couple of functions. Perhaps move it up?